lib/runnable.js

```diff
@@ -208,6 +209,8 @@ Runnable.prototype.run = function(fn){
 
   // finished
   function done(err) {
+    if (domain.active) { domain.active.exit(); }
```

Comment:
Entirely, it's my personal pref, but I can remove and re-submit if that'll match the style of the repo.